staging/src/k8s.io/kubelet/pkg/apis/dra-health/v1alpha1/api.proto

```diff
@@ -0,0 +1,42 @@
+// Copyright 2025 The Kubernetes Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+package v1alpha1;
+option go_package = "k8s.io/kubelet/pkg/apis/dra-health/v1alpha1";
+
+service ResourceHealth {
+  rpc WatchResources(WatchResourcesRequest) returns (stream WatchResourcesResponse) {}
```

Comment:
Does this run on nodes? Should we preprend `Node` to the method name?

Reply:
is it for alignement with other DRA methods? This is not the general rule ([example](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kubelet/pkg/apis/pluginregistration/v1/api.proto)), but I see your point, makes sense